decoder/pokemon.go

```diff
@@ -267,13 +262,34 @@ func savePokemonRecordAsAtTime(ctx context.Context, db db.DbDetails, pokemon *Po
 				pokemon.Pvp = null.StringFrom(string(pvpBytes))
 				changePvpField = true
 			}
+			var displayPokemon int64
```

Comment:
Why is this here in the pvp calculation and not where pokemon id is changed (ie when the wild or nearby is seen?)
Perhaps I am missing something.

The PVP calculation is only in the update so we don't persist it in the in-memory cache as it is so big, but this calculation (aside from using an ohbem function ) appears unrelated

Reply:
Hmm okay.